tests/main/docker-smoke/task.yaml

```diff
@@ -1,8 +1,8 @@
 summary: Check that the the docker snap works basically
 
 # only run on ubuntus for now, the docker snap has issues on non-ubuntu ATM
-systems:
-  - ubuntu-*
+# TODO:UC20: enable for UC20
+systems: [ubuntu-1*, ubuntu-2*, ubuntu-core-1*]
```

Comment:
shouldn't be needed as main suite is not enabled yet for uc20, right?

Reply:
That's what I thought too - but it turns out that spread will include ubuntu-core-20 here because the wildcard is more specific than the exclude pattern in the spread.yaml. So I had to add this or it would run. You can test this reverting this change and then run:
```
$ spread -v -list qemu:ubuntu-core-20|grep main
```

Reply:
is this test broken on uc20 ? just curious how it fails

Reply:
I honestly don't know yet :) I disabled the main test just so that I don't have to chase too many dragon. We will enable the full main suite really soon though.